### ISSUE TITLE
Props in docs?

### ISSUE THREAD

**Issue by Noitidart** [lesliesam/react-native-wheel-picker#34]
Hi there, may you please share the props in the readme?

**Comment by jenskuhrjorgensen** [comment]
@lesliesam I agree with @Noitidart - could you please provide a list of available props? :) This component looks awesome, but if I can't find out how to use it, it is no good to me.
